src/components/PeoplePage/PeoplePage.tsx

```diff
@@ -0,0 +1,67 @@
+import React, { useState, useEffect, memo } from 'react';
+
+import { Loader } from '../Loader/Loader';
+import { PeopleTable } from '../PeopleTable/PeopleTable';
+import { getPeople } from '../../api';
+import { Person } from '../../types/Person';
+
+export const PeoplePage: React.FC = memo(() => {
+  const [people, setPeople] = useState<Person[]>([]);
+  const [isLoading, setIsLoading] = useState(false);
+  const [isError, setIsError] = useState(false);
+  const [isLoadingFinish, setIsLoadingFinish] = useState(false);
```

Comment:
Why do you need this state at all? You can operate with `isLoading` state, and just check if loading was finished.
`isLoading` -> `true` - means not finished
`isLoading` -> `false` - finished